README.md

```diff
@@ -1,2 +1,20 @@
-# hw01
-Repository for first CFSS homework assignment
+# Cheng Yee Lim
+
+![](images/profile_pic.jpg)
+
+> *“A ship is safe in harbor, but that's not what ships are for.”*
+> ― **William G.T. Shedd**
+
+* Singaporean with a severe case of wanderlust
+  + Lived in the United Kingdom for the past three years 
+  + First year in the United States of America
+* Economist by training, but interested to pick up data analytics  
+* To find out more about the coding I do, click [here](www.github.com/limchengyee)
+
+## Process of Editing README.md 
+I first forked my own homework repository from the UC-CFSS. Then, I cloned my repository as a new project in RStudio. Thereafter, I began editing the README.md to include information about myself. Then, I modified the formatting, by including headers, blockquotes, bolding, and italicizing. Whenever I added a new component in, such as a picture, or the blockquote, I made a new commit and pushed it onto GitHub. This allowed me to track how my code developed and allows me to backtrack easily on GitHub. Lastly, I submitted a pull request to merge the changes on my repository (i.e. the modified README.md and the images folder) with the UC-CFSS homework repository. 
+
+##Reflections 
+The process of submitting my Rwork onto Git was smooth, as I have previously used GitHub for assignment submissions. However, I have mainly been using the command line for the submissions. It was interesting to how RStudio has an in-built Git function, which allows its users to upload their Rwork easily onto Git, facilitating collaboration among their uesrs. 
+
+It was, however, my first time writing with RMD. I used the [RMarkdown tutorial](http://rmarkdown.rstudio.com/lesson-1.html) to learn the syntax and capabilities of RMarkdown. I had two problems formatting the image - resizing and aligning. I solved the former by using html code to dictate the image size, but the html command for aligning it to the middle did not work. But I was not sure if we could include html code in our RMarkdown file in our assignment, so in the end, I used the usual way to add an image in the README file. 
```

Comment:
Markdown has no built in options to resize or align images. HTML can be used to resize, not sure why it didn't work for aligning as well